lib/vagrant-openshift/action/install_origin_base_dependencies.rb

```diff
@@ -215,6 +215,12 @@ def call(env)
   sudo lvcreate -n docker-data -l 90%FREE /dev/${VG}
   sudo lvcreate -n docker-metadata -l 50%FREE /dev/${VG}
   sudo sed -i "s,^DOCKER_STORAGE_OPTIONS=.*,DOCKER_STORAGE_OPTIONS='-s devicemapper --storage-opt dm.datadev=/dev/${VG}/docker-data --storage-opt dm.metadatadev=/dev/${VG}/docker-metadata'," /etc/sysconfig/docker-storage
+
+  sudo lvcreate -n openshift-test-tmp -l 100%FREE /dev/${VG}
```

Comment:
Should xfs be in the name?   It seems more specific than just openshift test tmp.